Scenarios/AKS-Secure-Baseline-PrivateCluster/Apps/SimpleGMSAApp/manifests/deployment_sampleapp.yml

```diff
@@ -74,12 +47,13 @@ spec:
               - key: agentpool
                 operator: In
                 values:
-                - wnp
+                - < Windows Nodepool Name >
```

Comment:
You can probably go back to hardcoding this, since you hardcoded setting it -- easier for the user to be successful.  I would suggest also adding in a windows OS match here as well, just to be explicit about requirements.